doajtest/fixtures/article_doajxml.py

```diff
@@ -1,11 +1,14 @@
 import os
+from io import BytesIO, StringIO
+from pathlib import Path
+
 from lxml import etree
-from io import BytesIO,StringIO
-from copy import deepcopy
 
 RESOURCES = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "unit", "resources")
 ARTICLES = os.path.join(RESOURCES, "doajxml_article_uploads.xml")
 
+ARTICLE_UPLOAD_SUCCESSFUL = Path(os.path.join(RESOURCES, "article_upload_successful.xml")).resolve().as_posix()
```

Comment:
We already have a library for path management `portality.lib.paths`, so I suggest that we do the same here as we have with dates, and ensure all path manipulation is done in there.  We can still import `pathlib` there, but lets keep all that stuff encapsulated

Reply:
Yeah, I was just imagining adding a function to `portality.lib.paths` like this

```python
def posix_path(source_file):
    return Path(os.path.join(source_dir)).resolve().as_posix()
```

But you make a good point that we should refactor this across the board.  Could you open a refactoring issue for that, then this is fine to leave as is